docs/gatsby-config.js

```diff
@@ -19,41 +19,40 @@ module.exports = {
           null: [
             'index',
             'getting-started',
+            'graphql-playground',
             '[Changelog](https://github.com/apollographql/apollo-server/blob/master/CHANGELOG.md)',
           ],
           'Defining a Schema': [
-            'defining-a-schema/schema',
-            'defining-a-schema/scalars-enums',
-            'defining-a-schema/unions-interfaces',
-            'defining-a-schema/directives',
-            'defining-a-schema/creating-directives',
+            'schema/schema',
+            'schema/scalars-enums',
+            'schema/unions-interfaces',
+            'schema/directives',
+            'schema/creating-directives',
           ],
-          'Resolving Requests': [
-            'resolving-requests/data',
-            'resolving-requests/data-sources',
-            'resolving-requests/errors',
-            'resolving-requests/authentication',
-            'resolving-requests/file-uploads',
-            'resolving-requests/subscriptions',
+          'Serving Requests': [
+            'serving/data',
+            'serving/data-sources',
+            'serving/errors',
+            'serving/authentication',
+            'serving/file-uploads',
+            'serving/subscriptions',
           ],
-          'Testing & Monitoring': [
+          Testing: [
             'testing-and-monitoring/mocking',
-            'testing-and-monitoring/graphql-playground',
             'testing-and-monitoring/testing',
-            'testing-and-monitoring/metrics',
-            'testing-and-monitoring/health-checks',
-          ],
-          'Improving Performance': [
-            'improving-performance/caching',
-            'improving-performance/apq',
           ],
+          'Improving Performance': ['performance/caching', 'performance/apq'],
           Deployment: [
             'deployment/heroku',
             'deployment/lambda',
             'deployment/now',
             'deployment/netlify',
             'deployment/azure-functions',
           ],
+          Monitoring: [
+            'testing-and-monitoring/metrics',
```

Comment:
Similarly to above, perhaps this should be just in a `monitoring/` directory now?